client/js/options.js

```diff
@@ -91,6 +93,13 @@ settings.on("change", "input, select, textarea", function() {
 			// otherwise, users get notifications for everything
 			return h !== "";
 		});
+		// Construct regex with wordboundary for every highlight item
+		var highlightsTokens = options.highlights.map(function(h) {
```

Comment:
You mean something like this?
```
options.highlights = highlightString.split(",").map(function(h) {
  return escapeRegExp(h.trim());
})
```
Cause that would mean the contents of options.highlight will be escaped. I was afraid that might break other stuff that expected it to not be escaped.